actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/javadsl/Junit5TestKit.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, derived from Akka.
```

Comment:
Could you rebase this PR? Today, I've changed the license headers in most of our files and you should us the new header.

Line 7 here is replaced with:
```
This file is part of the Apache Pekko project, which was derived from Akka.
```